src/schema/v2/system/userRoles.ts

```diff
@@ -11,7 +12,7 @@ const UserRoleType = new GraphQLObjectType<any, ResolverContext>({
   description: "Fields corresponding to a given product privilege",
   fields: {
     name: {
-      type: GraphQLString,
+      type: new GraphQLNonNull(GraphQLString),
```

Comment:
Note that if this in anyway _can_ be null or undefined, it will error out at the MP level. So we only mark as non-nullable if we're absolutely certain that there is a value upstream from here. If not sure, better to be safe and leave it alone, and deal with it at the client consumer level.